cmd/bank/main.go

```diff
@@ -31,60 +129,619 @@ func main() {
 		panic(err)
 	}
 
-	messages := []dogma.Message{
+	as := appState{
+		db:     db,
+		engine: en,
+		reader: bufio.NewReader(os.Stdin),
```

Comment:
This reader should be removed. Its from the first text UI design and is no longer used.